packages/databases-collections/src/modules/create-namespace.ts

```diff
@@ -100,9 +100,13 @@ export const open = (
   dbName: string | null = null
 ): CreateNamespaceThunkAction<void, OpenAction> => {
   return (dispatch, _getState, { track }) => {
-    track('Screen', {
-      name: dbName ? 'create_collection_modal' : 'create_database_modal',
-    });
+    track(
+      'Screen',
+      {
+        name: dbName ? 'create_collection_modal' : 'create_database_modal',
+      },
+      undefined
+    );
```

Comment:
fyi: These are the events I skipped before as we said we probably don't need the connectionId in all the screen events, but if we wanted to we could get it from connectionRepository. similar to -> https://github.com/mongodb-js/compass/blob/980e5a27dda173940df4e8abf557fabb9d1b782a/packages/databases-collections/src/modules/create-namespace.ts#L403